network/stations.csv

```diff
@@ -940,6 +940,7 @@ EXPV,XX,Express,-39.253545373,175.564881983,2030,,WGS84,1995-10-25T03:58:00Z,199
 EYCZ,CB,Eyrewell,-43.42136,172.35544,149.7,,WGS84,2012-03-20T00:00:00Z,2013-12-03T00:00:00Z
 EYRM,GM,Eyrewell Geomagnetic Observatory,-43.42134,172.35544,115,,WGS84,1978-01-01T00:00:00Z,2013-12-15T23:59:59Z
 EYRS,SM,Eyrewell Observatory,-43.42138,172.35544,119,,WGS84,2011-03-25T00:00:00Z,2012-10-15T10:45:00Z
+EY2M,GM,West Melton 2 Geomagnetic Observatory,-43.47384,172.39266,102,,WGS84,2024-04-04T16:00:00Z,9999-01-01T00:00:00Z
```

Comment:
- please change the name to `West Melton Geomagnetic Observatory 2` (or keep as is)
- the height here is 102, and in sites is 88. what is the true one? we shall use the same

Reply:
I think we'll stick with West Melton 2 Geomagnetic Observatory - I'll amend other locations accordingly.
I copied height from existing West Melton site - I'll amend both to the correct height of 88 in sites.csv